community.json

```diff
@@ -44,6 +44,13 @@
         "state": "inprogress",
         "url": "https://github.com/YunoHost-Apps/agora_ynh"
     },
+    "airsonic": {
+        "branch": "master",
+        "level": 3,
```

Comment:
The level is fixed by the CI. And the CI test the app only if the app is tagged as "working"